docs/pages/reference/machine-id/bound-keypair.mdx

```diff
@@ -0,0 +1,567 @@
+---
+title: Bound Keypair Joining
+description: "Bound Keypair Joining: Reference and admin guide"
+---
+
+Bound Keypair is a new join method designed to provide the best features of
+[dynamic join methods][dynamic] join methods - like AWS, GCP, or Azure - but in
+on-prem or otherwise unsupported environments where no external verification is
+available.
+
+Specifically, this join method:
+- Does not require dedicated TPM hardware or external identity attestation
+- Does not require long-lived shared secrets
+- Allows for limited automatic recovery if certificates expire
+- Allows recovery restrictions to be relaxed or lifted to accommodate different
+  use cases and deployment scenarios
+- Ensures failed bots can be recovered without client-side intervention in most
+  cases
+
```

Comment:
I would be clearer about what readers would expect in this guide. The introduction frames it as a general-purpose docs page on bound keypair joining, but that puts it in conflict with the how-to guide. I would consider:
- Splitting the architectural information into a separate guide with a more specific title and putting that in the Architecture section.
- Retitling this page to be more specific, e.g., Bound Keypair Joining Reference.
- Making sure each Bound Keypair Joining guide links to the other Bound Keypair Joining guides in the introduction.

Reply:
I like option 2.

Reply:
I've at least retitled the page and added a guides section to link to the main deployment guide. I'll admit it definitely doesn't fit cleanly into "just" being a reference page and is sort of a cursed hybrid of an architecture page, admin manual, and configuration reference.

I'm not opposed making additional separate pages, but I think (for example) a dedicated architecture page might be a bit light on content given the "Concepts" section as it exists now. Definitely happy to hear your thoughts on the best path forward!